src/raft/raft.rs

```diff
@@ -1624,7 +1633,7 @@ impl<T: Storage> Raft<T> {
 
         // The quorum size is now smaller, so see if any pending entries can
         // be committed.
-        if self.maybe_commit() {
+        if self.maybe_commit() && !self.skip_bcast_commit {
```

Comment:
This doesn't happen frequently, whether check it doesn't make much difference. But as configuration name says, when it's enabled, broadcast of commit should be skipped.